vendor-support/pom.xml

```diff
@@ -32,6 +32,13 @@
         <dependency>
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-java</artifactId>
+            <version>${selenium.version}</version>
```

Comment:
I don't understand what this is doing. It appears to be both including and excluding at the same time.